generic/tests/cfg/nfs_corrupt.cfg

```diff
@@ -16,9 +16,8 @@
     post_command_noncritical = yes
     wait_paused_timeout = 240
     nfs_stat_chk_re = "running"
-    s390x:
-        bootindex_image1 = 0
-        bootindex_stg = 1
+    serial = TARGET_DISK
+    blk_extra_params_stg += "serial=${serial}"
```

Comment:
For `+=` operation, you should use `,serial=${serial}`, an example is `blk_extra_params_stg = key=value` + `blk_extra_params_stg += "serial=${serial}"`, it will become `blk_extra_params_stg = key=valueserial=${serial}"`. If qcontainer cannot handle it, it's bug of qcontainer, or you have to use `=` operation here.

Reply:
agree

Reply:
updated